packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -280,13 +292,16 @@ const generateComponentMeta = (): void => {
     // breakpointCustomizableProps
     const breakpointCustomizableProps: ComponentMeta['breakpointCustomizableProps'] = [];
 
+    // deprecatedPropValues
```

Comment:
does this comment bring any additional value? It just repeats the constant's name...

Reply:
Individually it doesn't help, but looking at the entire file you see that those are basically markers for a better overview.